dadi/lib/controller/media.js

```diff
@@ -288,6 +297,7 @@ MediaController.prototype.delete = function (req, res, next) {
     storageHandler.delete(file)
       .then(result => {
         this.model.delete({
+          client: req.dadiApiClient,
```

Comment:
@eduardoboucas do you think we should check they have DELETE permission before removing the physical file?

Reply:
YES! Good spot, thank you.